README.md

```diff
@@ -56,6 +56,14 @@
 
 .NET for Apache Spark releases are available [here](https://github.com/dotnet/spark/releases) and NuGet packages are available [here](https://www.nuget.org/packages/Microsoft.Spark).
 
+### First-Party Extensions
+Some extended features are available via extension NuGet packages:
+* [Microsoft.Spark.Extensions.Delta](https://www.nuget.org/packages/Microsoft.Spark.Extensions.Delta/)
```

Comment:
Shall we put a short description for each package? For example, ".NET binding for Delta Lake" (with a link to delta lake page)